plugin/trino-oracle/src/main/java/io/trino/plugin/oracle/OracleClient.java

```diff
@@ -204,7 +204,7 @@ protected String generateTemporaryTableName()
     protected void renameTable(ConnectorSession session, String catalogName, String schemaName, String tableName, SchemaTableName newTable)
     {
         if (!schemaName.equalsIgnoreCase(newTable.getSchemaName())) {
-            throw new TrinoException(NOT_SUPPORTED, "Table rename across schemas is not supported in Oracle");
+            throw new TrinoException(NOT_SUPPORTED, "This connector does not support renaming tables across schemas");
```

Comment:
Could you please doublecheck if exception message corresponds to condition correctly. 
I don't understand how case equality correlates with `across schemas`.

Reply:
I had to add tests for `renameTable` since this is where the issue with schema name case manifested itself. The tests for `createSchema` only are not enough, since `getSchemaNames` already return lower-cased names and I could figure out other asserts.

When working on rename table tests I noticed inconsistencies in how connectors report that it's not supported and fixed that, hence this change.